lib/src/Uncertainty/Algorithm/OrthogonalBasis/openturns/OrthogonalProductFunctionFactory.hxx

```diff
@@ -62,12 +62,18 @@ public:
   /** Build the Function of the given index */
   Function build(const UnsignedInteger index) const override;
 
+  /** Build the Function of the given multi-indices */
+  Function build(const Indices & indices) const;
```

Comment:
this should be override here too (and virtual in the parent)

Reply:
Fixed.

Reply:
Fixed.